subworkflows/encyclopedia.nf

```diff
@@ -77,9 +115,9 @@ workflow PERFORM_QUANT {
             // Output is [group, global_elib_file, peptides_txt, proteins_txt, log]
             ENCYCLOPEDIA_GLOBAL(
                 local_files,
-                dlib,
                 fasta,
-                params.encyclopedia.quant_postfix
+                params.encyclopedia.quant_postfix,
+                true
```

Comment:
What is this true/false variable? Is there a way we can assign it in the parameters instead of specifying it here?